Packs/RiskIQDigitalFootprint/Integrations/RiskIQDigitalFootprint/TestData/get_asset_contact_custom_ec.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "domains@riskiq.com",
+  "type": "CONTACT",
+  "firstSeen": 1474994431102,
+  "lastSeen": 1591634267273,
+  "uuid": "3ae11a55-2925-c271-7806-f0839ce24350",
+  "label": "domains@riskiq.com",
+  "description": "domains@riskiq.com",
+  "assetEmail": "domains@riskiq.com",
+  "assetNames": [
+    {
+      "value": "Elias Manousos",
```

Comment:
Are you positive that all of the example data you've added for your unit tests has been cleaned of any real response data that may or may not include, individuals' personal data and/or personal or corporate sensitive information? I ask again only because I noticed this name, which seems to be a real person, and it's very important that we don't add the aforementioned types of data to the repository.

Reply:
@avidan-H We confirmed with our client, he was fine with it. But yes, we shall make the changes in terms of names and contacts, so we will be adding fake data for such cases.